### DISCUSSION TITLE
exios error

### DISCUSSION THREAD

**Discussion by 001Jannat** [axios/axios#6095] [Q&A]
xhr.js:256 
 POST http://localhost:3000/api/candidate 401 (Unauthorized)
app-index.js:32 Error adding candidate: 
AxiosError {message: 'Request failed with status code 401', name: 'AxiosError', code: 'ERR_BAD_REQUEST', config: {…}, request: XMLHttpRequest, …}
code
: 
"ERR_BAD_REQUEST"
config
: 
{transitional: {…}, adapter: Array(2), transformRequest: Array(1), transformResponse: Array(1), timeout: 0, …}
message
: 
"Request failed with status code 401"
name
: 
"AxiosError"
request
: 
XMLHttpRequest {onreadystatechange: null, readyState: 4, timeout: 0, withCredentials: false, upload: XMLHttpRequestUpload, …}
response
: 
{data: {…}, status: 401, statusText: 'Unauthorized', headers: AxiosHeaders, config: {…}, …}
stack
: 
"AxiosError: Request failed with status code 401\n    at settle (webpack-internal:///(app-pages-browser)/./node_modules/axios/lib/core/settle.js:24:12)\n    at XMLHttpRequest.onloadend (webpack-internal:///(app-pages-browser)/./node_modules/axios/lib/adapters/xhr.js:126:66)"
[[Prototype]]
: 
Error

**Comment by LinThitHtwe** [comment]
It appears that your request is encountering a 401 error, suggesting a lack of authorization. Please ensure that you have included your token in the request.
